docs/Using Randomness/example-contracts.md

```diff
@@ -25,23 +25,22 @@ How you manage the subscription depends on your randomness needs. You can config
 
 ## Modifying subscriptions and configurations
 
-Subscription configurations do not have to be static. You can change your subscription configuration dynamically by calling the following functions on the VRF v2 coordinator contract:
+Subscription configurations do not have to be static. You can change your subscription configuration dynamically by calling the following functions using the [VRFCoordinatorV2Interface](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/interfaces/VRFCoordinatorV2Interface.sol):
 
-- Change the consumer set with `addConsumer(uint64 subId, address consumer)` or `removeConsumer(uint64 subId, address consumer)`.
+- Change the list of approved subscription consumers with `addConsumer(uint64 subId, address consumer)` or `removeConsumer(uint64 subId, address consumer)`.
 - Transfer the subscription ownership with `requestSubscriptionOwnerTransfer(uint64 subId, address newOwner)` and `acceptSubscriptionOwnerTransfer(uint64 subId)`.
-- Top up the subscription balance with `LINKTOKEN.transferAndCall(address(COORDINATOR), amount, abi.encode(subId));`. Any wallet can fund a subscription.
 - View the subscription with `getSubscription(uint64 subId)`.
 - Cancel the subscription with `cancelSubscription(uint64 subId)`.
 
-The full coordinator interface is available on [GitHub](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/interfaces/VRFCoordinatorV2Interface.sol). You can use the subscription management functions however you see fit.
+To send LINK to the subscription balance, use the LINK token interface with `LINKTOKEN.transferAndCall(address(COORDINATOR), amount, abi.encode(subId));`. Any wallet can fund a subscription.
 
 See the example in the [Subscription manager contract](#subscription-manager-contract) section to learn how to create a contract that can change your subscription configuration.
 
 ## Subscription manager contract
 
-In this example, the contract operates as a subscription owner and can run functions to add consumer contracts to the subscription. Those contracts need to include only the `requestRandomWords()` function, the `fulfillRandomWords()` functions with the correct coordinator parameters and the correct `subscriptionId` value to obtain their own random values and use the subscription balance.
+In this example, the contract operates as a subscription owner and can run functions to add consumer contracts to the subscription. Those contracts need to include only the `requestRandomWords()` function, the `fulfillRandomWords()` functions with the correct coordinator parameters and the correct subscription ID to obtain their own random values and use the subscription balance.
```

Comment:
@dwightjl we should remove _Only_ . This can be confusing for the reader as he thinks he has to only include _requestRandomWords()_ in the consumer contract.
Moreover,  this sentence "with the correct coordinator parameters and the correct subscription ID" is only relevant for _requestRandomWords()_ . In fact, _fulfillRandomWords()_ is the callback function and doesn't need to handle coordinator parameters.

I will create another PR to complete "What's Next" and add the promotional video & blog. I can fix this in this PR as well